examples/cmd/decrypt.go

```diff
@@ -40,42 +39,38 @@ func decrypt(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+
 	defer file.Close()
+	cmd.Println("# TDF")
 
-	var magic [3]byte
-	var isNano bool
-	n, err := io.ReadFull(file, magic[:])
-	switch {
-	case err != nil:
-		return err
-	case n < 3:
-		return errors.New("file too small; no magic number found")
-	case bytes.HasPrefix(magic[:], []byte("L1L")):
-		isNano = true
-	default:
-		isNano = false
-	}
-	_, err = file.Seek(0, 0)
+	var isTDF = true
+	tdfreader, err := client.LoadTDF(file)
```

Comment:
aw, I miss my magic header check. I was kinda hoping to move this into sdk, so maybe we should move this to an issue saying we dshould have a unified API where nano/whatever is an option instead of having separate methods?

Reply:
SDK function to detect isTDF() and to detect the format https://github.com/opentdf/platform/issues/1000